cmd/amqpproxy/main_test.go

```diff
@@ -67,6 +67,32 @@ func TestAMQPProxy(t *testing.T) {
 	testhelpers.ValidateLog(t, testData.JSONLFile)
 }
 
+func TestAMQPProxyExcludePayloadData(t *testing.T) {
+	testEnv.SkipIfNotLive(t)
+
+	testData := mustCreateAMQPProxyExcludePayloadData(t, []string{})
+
+	receiver, err := testData.ServiceBusClient.NewReceiverForQueue(testData.ServiceBusQueue, nil)
```

Comment:
`receiver` is never actually used, but probably should be for this test so we can guarantee that incoming TRANSFER frames are also redacted.